frontend/components/EnrollSecrets/EnrollSecretModal/EnrollSecretModal.tsx

```diff
@@ -85,9 +85,9 @@ const EnrollSecretModal = ({
           <GitOpsModeTooltipWrapper
             position="right"
             tipOffset={8}
-            renderChildren={(disableChildren) => (
+            renderChildren={(dC) => (
```

Comment:
Curious why concatenating `disableChildren` to `dC`? React pattern is to spell words out, Go is to abbreviate. Kinda confusing when moving between the two!

Reply:
Great point, I'll change these to `disableChildren`

Reply:
It's funny how our backend has the opposite opinion as our frontend on variable brevity 😆 

PR looks good! This feature caught my interest because this has bit me several times.